docs/en/relying-party-solution.rst

```diff
@@ -0,0 +1,308 @@
+.. include:: ../common/common_definitions.rst
+
+.. _relying-party-solution:
+
+Relying Party Solution
+++++++++++++++++++++++
+
+This document defines the implementation of the online presentation and verification flow of a credential (PID or (Q)EAA), in accordance with the "OpenID for Verifiable Credential Presentation - draft 19" specifications [OID4VCP], in the context of a Cross-Device flow scenario.
+
+
+Cross Device Authentication
+---------------------------
+
+In the Cross Device Authentication, the user logs into the website and provides their consent for presenting their PID through the Wallet. 
+Once authentication is performed, the end user can request the display of attributes provided by the Wallet.
+Below is explained the Cross Device Authentication flow.
+
+NOTE: Use Cases about other flows will be analysed in next sprints.
```

Comment:
sprints -> releases of this technical specification

Reply:
Done